### ISSUE TITLE
Option status inside a selector not read when "selected"

### ISSUE THREAD

**Issue by emaestromartinez** [nvaccess/nvda#16449]
### Steps to reproduce:
Use angular material Select component as an example.
https://v7.material.angular.io/components/select/overview#multiple-selection
Navigate through the list using keyboard navigation. Have some options selected and others unselected.

### Current behavior:
When using the screenreader to read the selected/unselect, NVDA doesn't announce the 'Selected' state for selected list item when user navigates on list items.
NVDA only announces list item and its position in the list.
It does announce when they are selected.

### Expected behavior:
It should also announce "Selected" for selected items, not just the name and position on the list.

#### NVDA version:
Last stable version.

#### Windows version:
Microsoft Windows 10 Enterprise. Last stable version.

**Comment by seanbudd** [comment]
Welcome @emaestromartinez ,

We are unable to process this issue as it stands.

Please fill out the issue template fully. You need to include the version numbers of the browser, NVDA and Windows you are using to reproduce the issue.

Kind Regards,
NV Access Software Developers

**Comment by SaschaCowley** [comment]
This is the established behaviour in NVDA. For example, if you open File Explorer, make a selection, then move without changing the selection (Ctrl+Arrows), NVDA will report "not selected" for items that are not selected, but will not make a special announcement for items that are selected. I would suggest you open a new issue if you believe this behaviour should be changed.
